src/m365/pp/commands/dataverse/dataverse-table-row-list.spec.ts

```diff
@@ -0,0 +1,148 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { powerPlatform } from '../../../../utils/powerPlatform';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+const command: Command = require('./dataverse-table-row-list');
+
+describe(commands.DATAVERSE_TABLE_ROW_LIST, () => {
+  //#region Mocked Responses
+  const validEnvironment = '4be50206-9576-4237-8b17-38d8aadfaa36';
+  const validName = 'cr6c3_clitables';
+  const envUrl = "https://contoso-dev.api.crm4.dynamics.com";
+  const rowsResponse = {
+    "value": [
+      {
+        "cr6c3_clitablesid": "95c80273-3764-ed11-9561-000d3a4bbea4",
+        "_owningbusinessunit_value": "6da087c1-1c4d-ed11-bba1-000d3a2caf7f",
+        "statecode": 0,
+        "statuscode": 1,
+        "_createdby_value": "5fa787c1-1c4d-ed11-bba1-000d3a2caf7f",
+        "_ownerid_value": "5fa787c1-1c4d-ed11-bba1-000d3a2caf7f",
+        "modifiedon": "2022-11-14T16:14:45Z",
+        "_owninguser_value": "5fa787c1-1c4d-ed11-bba1-000d3a2caf7f",
+        "_modifiedby_value": "5fa787c1-1c4d-ed11-bba1-000d3a2caf7f",
+        "versionnumber": 1413873,
+        "createdon": "2022-11-14T16:14:45Z",
+        "cr6c3_columnone": "Test1",
+        "overriddencreatedon": null,
+        "importsequencenumber": null,
+        "_modifiedonbehalfby_value": null,
+        "utcconversiontimezonecode": null,
+        "_createdonbehalfby_value": null,
+        "_owningteam_value": null,
+        "timezoneruleversionnumber": null
+      }
+    ]
+  };
+  //#endregion
+
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    auth.service.connected = true;
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    loggerLogSpy = sinon.spy(logger, 'log');
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get,
+      powerPlatform.getDynamicsInstanceApiUrl
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      appInsights.trackEvent,
+      pid.getProcessName
+    ]);
+    auth.service.connected = false;
+  });
+
+  it('has correct name', () => {
+    assert.strictEqual(command.name.startsWith(commands.DATAVERSE_TABLE_ROW_LIST), true);
```

Comment:
Like we did for `pp solution publisher add` can we get rid of the `startsWith` here?